aemedge/blocks/carousel/carousel.js

```diff
@@ -148,4 +149,20 @@ export default async function decorate(block) {
   if (!isSingleSlide) {
     bindEvents(block);
   }
+  // Auto-scrolling functionality
+  let slideIndex = 0;
+  const slides = block.querySelectorAll('.carousel-slide');
+
+  function autoScroll() {
+    slideIndex += 1;
+    if (slideIndex >= slides.length) {
+      slideIndex = 0;
+    }
+    showSlide(block, slideIndex);
+  }
+
+  // Call autoScroll every 3 seconds
+  if (variant.includes('medium')) {
+    setInterval(autoScroll, 3000);
```

Comment:
Can you call autoScroll every 5 seconds? And can you also make the transition slower?